<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseKeys</class>
 <widget class="QDialog" name="ChooseKeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="save_as">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save as</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="quit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="save_as_keys">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save keys</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="save_keys">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="open_keys">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="quit_keys">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
